src/structures/NewsChannel.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const TextChannel = require('./TextChannel');
+
+/**
+ * Represents a guild news channel on Discord.
+ * @extends {TextChannel}
+ */
+class NewsChannel extends TextChannel {
+  _patch(data) {
+    super._patch(data);
+
+    // News channels don't have a rate limit per user, remove it
+    this.rateLimitPerUser = undefined;
```

Comment:
I think 0 would be more suitable, as it means it's unset.

Reply:
The payload does not provide a rate_limit_per_user
```json
{
    "id": "41771983423143937",
    "guild_id": "41771983423143937",
    "name": "important-news",
    "type": 5,
    "position": 6,
    "permission_overwrites": [],
    "nsfw": true,
    "topic": "Rumors about Half Life 3",
    "last_message_id": "155117677105512449",
    "parent_id": "399942396007890945"
}
```

While I agree it'd keep things consistent, it wouldn't be 100% accurate with the API docs

Reply:
Stop emailing me